detectable/src/main/java/com/synopsys/integration/detectable/detectables/bitbake/parse/BitbakeRecipesParser.java

```diff
@@ -78,4 +67,13 @@ private BitbakeRecipe parseLine(String line, BitbakeRecipe currentRecipe, Map<St
             return currentRecipe;
         }
     }
+
+    private void finishCurrentRecipe(final Map<String, List<String>> bitbakeRecipes, final Set<String> layerNames, final BitbakeRecipe currentRecipe) {
```

Comment:
@stevebillings I think your Save Action settings are incorrect. They should be removing final here.
Also we should add the `@Nullable` annotation here.
```suggestion
    private void finishCurrentRecipe(Map<String, List<String>> bitbakeRecipes, Set<String> layerNames, @Nullable BitbakeRecipe currentRecipe) {
```

Reply:
Darn. Should be fixed now.